pkg/artifactory/resource_artifactory_virtual_helm_repository.go

```diff
@@ -0,0 +1,46 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceArtifactoryHelmVirtualRepository() *schema.Resource {
+
+	helmVirtualSchema := mergeSchema(baseVirtualRepoSchema, map[string]*schema.Schema{
+		"use_namespaces": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "From Artifactory 7.24.1 (SaaS Version), you can explicitly state a specific aggregated local or remote repository to fetch from a virtual by assigning namespaces to local and remote repositories\nSee https://www.jfrog.com/confluence/display/JFROG/Kubernetes+Helm+Chart+Repositories#KubernetesHelmChartRepositories-NamespaceSupportforHelmVirtualRepositories",
```

Comment:
Can we mention the default value and (Optional) in description text? To maintain the convention.

Reply:
Agree

Reply:
Good catch!